src/components/CampaignContactsChoiceForm.jsx

```diff
@@ -72,23 +72,52 @@ export class CampaignContactsChoiceForm extends React.Component {
 
   handleChange(contactData) {
     this.props.onChange({
-      contactData: contactData,
+      contactData,
       ingestMethod: this.getCurrentMethod().name
     });
   }
 
   render() {
-    const { ingestMethodChoices, pastIngestMethod } = this.props;
+    const {
+      maxNumbersPerCampaign,
+      contactsPerPhoneNumber,
+      ingestMethodChoices,
+      pastIngestMethod
+    } = this.props;
     const ingestMethod = this.getCurrentMethod();
     const ingestMethodName = ingestMethod && ingestMethod.name;
     const lastResult =
       pastIngestMethod && pastIngestMethod.name === ingestMethodName
         ? pastIngestMethod
         : null;
     const IngestComponent = components[ingestMethodName];
+
     return (
       <div>
         <CampaignFormSectionHeading title="Who are you contacting?" />
+
+        {contactsPerPhoneNumber && maxNumbersPerCampaign && (
+          <div
+            style={{
+              marginBottom: 10,
+              fontSize: 17,
+              color: theme.colors.darkBlue
+            }}
+          >
+            <div>
+              You can only upload a max of{" "}
+              {Number(
+                contactsPerPhoneNumber * maxNumbersPerCampaign
+              ).toLocaleString()}{" "}
+              contacts per campaign.
+            </div>
+            <div>
+              Each campaign be assigned {maxNumbersPerCampaign} numbers max with{" "}
```

Comment:
"Each campaign **can** be assigned" -- missing "can"?

Reply:
yep thanks lol